Introduction to Simulation.ipynb

```diff
@@ -6,12 +6,43 @@
    "source": [
```

Comment:
<blockquote>With these elements, you can predict the position and velocity of the ball at any point in time...<strong>will come back to this at the end of the presentation.</strong>  </blockquote><p>Is the last section meant to be there? (is it just meant to be presented but not written, or both)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PTC-CMC/cybercamp/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>We actually code this simulation up later.  I've revised the text to be clearer.</p><p> </p>